roles/setup_efm/tasks/setup_efm.yml

```diff
@@ -116,6 +116,20 @@
   include_tasks: efm_cluster_set_params.yml
   no_log: "{{ disable_logging }}"
 
+- name: Gather service facts
+  service_facts:
+
+- name: Open EFM TCP ports 7800-7810
+  firewalld:
+    port: "7800-7810/tcp"
```

Comment:
Shall we use a variable instead of hardcoded port? Let's add a variable efm_port in defaults/main.

Reply:
Please see this new commit 597f3f1